### PR TITLE
FIX: Ridge regressor - L2 regularization formula on notebook 4

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
The formula in the notebook seems incorrect.
Comparing to https://scikit-learn.org/stable/auto_examples/linear_model/plot_ridge_coeffs.html

### CODE REVIEW

File: content/python_files/04_linear_model_regularization.py [hunk 1]
```diff
@@ -132,7 +132,7 @@
 # L1 provides another regularization type. It follows this formula:
 #
 # $$
-# loss = (y - X \beta)^2 + \alpha \|\beta\|_1
+# loss = \frac{1} {2 n\_samples} ||y - X\beta||^2_2 + \alpha  ||\beta||_1
```
Comment (glemaitre) [inline content/python_files/04_linear_model_regularization.py]: I think that we can omit the normalization factor for all the losses to have the two losses consistent. Basically, the literature should show both formula.

```suggestion
# loss = ||y - X\beta||^2_2 + \alpha  ||\beta||_1
```
